build/crd-samples/devices/CC2650-device-instance.yaml

```diff
@@ -1,5 +1,5 @@
 
-apiVersion: devices.kubeedge.io/v1alpha2
```

Comment:
Please revert these changes, thanks.

Reply:
ok

Reply:
The request is restricted by github，i will push later.